README.md

```diff
@@ -1,111 +1,39 @@
-```
-
-For this to work you need to run a Datadog agent that listens for APM traffic at `localhost:8126`. If your agent is reachable under a different address, you can modify the `-collection-agent` parameter accordingly.
-
-## Running inside a container
-
-#### Requirements
-
-When running the agent in a container, you need to ensure the following conditions are met:
-* The container is running in privileged mode.
-* The container has the `SYS_ADMIN` capability.
-* The container has Host PID enabled (and procMount: "Unmasked").
-* The host's debugfs filesystem is mounted to the container (in read-only mode).
-* The agent is running as root inside the container.
-
-#### Container name resolution
-
-To be able to resolve container names, the agent needs to be able to access the underlying container runtime (in read-only mode). The agent supports Docker and containerd.
+If you're not using a container runtime, please check this section to run the profiler directly on the host: [Running on the host](doc/running-on-host.md).
 
-To enable this feature, you need to mount the container runtime socket to the agent container in read-only mode (`/var/run/docker.sock` for Docker, `/run/containerd/containerd.sock` for containerd).
-
-#### Pod name resolution
-
-To be able to resolve pod names in Kubernetes, the agent needs to be able to:
-
-1. Get the `KUBERNETS_NODE_NAME` environment variable:
-```yaml
-env:
-  - name: KUBERNETES_NODE_NAME
-    valueFrom:
-      fieldRef:
-        fieldPath: spec.nodeName
-```
-
-2. Access the underlying Kubernetes API server. This is usually done through a ClusterRole and ClusterRoleBinding with the following permissions:
-```yaml
-rules:
-  - verbs:
-      - get
-      - watch
-      - list
-    resources:
-      - nodes
-      - pods
-    apiGroups:
-      - ""
-```
-
-## Configuration
+## Configuring the profiler
 
 ### Local symbol upload (Experimental)
 
-For compiled languages (C/C++/Rust/Go), the profiling-agent can upload local symbols (when available) to Datadog for symbolication. Symbols need to be available locally (unstripped binaries).
+For compiled languages (C/C++/Rust/Go), the profiler can upload local symbols (when available) to Datadog for symbolication. Symbols need to be available locally (unstripped binaries).
 
 To enable local symbol upload:
 1. Set the `DD_EXPERIMENTAL_LOCAL_SYMBOL_UPLOAD` environment variable to `true`.
 2. Provide a Datadog API key through the `DD_API_KEY` environment variable.
-3. Set the `DD_SITE` environment variable to [your Datadog site](https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site) (e.g. `datadoghq.com`).
-
+3. Set the `DD_SITE` environment variable to [your Datadog site](https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site) (e.g. `datadoghq.com`, `datadoghq.eu`, `us5.datadoghq.com`, ...).
```

Comment:
IIRC this only works for orgs that we explicitly whitelist right now? If yes, we should mention that this requires being part of our private beta and to contact support if they are interested?